docs/sources/tempo/operations/monitor/set-up-monitoring.md

```diff
@@ -12,45 +12,45 @@ If you don't have a cluster available, you can use the linked documentation to s
 
 To set up monitoring, you need to:
 
-* Use Grafana Agent Flow or Grafana Alloy to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/latest/setup/set-up-test-app/).
-* Update your Grafana Agent Flow configuration to scrape metrics to monitor for your Tempo data.
+* Use Grafana Alloy collector to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
```

Comment:
I'd either use `the Grafana Alloy collector` or drop the word `collector` here.

Reply:
```suggestion
* Use Grafana Alloy to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
```